heltour/tournament/templates/tournament/availability.html

```diff
@@ -47,12 +58,18 @@ <h3>Edit Availability</h3>
                                     <td>{{ r.start_date|date_or_q:'%b %-d' }} - {{ r.end_date|date_or_q:'%b %-d' }}</td>
                                     {% for p, is_av, is_immutable in av_list %}
                                         <td>
-                                            <input type="checkbox" name="av_r{{ r.number }}_{{ p.lichess_username }}"
+                                            <input id="availability-checkbox-{{ r.number }}_{{ p.lichess_username }}"
```

Comment:
I need to double check, but you might be able to use the django `{% with .. %}` tag to make this code a bit nicer: https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#with

Reply:
The idea is that I'm hoping something like this works:
```django
{% with id="{{ r.number }}_{{ p.lichess_username }}" %}
<input id="availability-checkbox-{{ id }}"
{% endwith %}
```

Reply:
Thanks, I will try that.